site2/docs/io-overview.md

```diff
@@ -34,9 +34,106 @@ Common sinks include other messaging systems and SQL and NoSQL databases.
 
 For the complete list of Pulsar built-in sink connectors, see [sink connector](io-connectors.md#sink-connector).
 
+## Processing guarantee
+
+Processing guarantees are used to:
+
+* handle errors when writing messages to Pulsar topics.
+  
+* determine the subscription type used by Pulsar when reading messages from Pulsar topics.
```

Comment:
I don't think this part of the content is necessary, and there is no necessary relationship between `Processing guarantees` and `subscription type`.

Reply:
@tuteng thank you. I've incorporated your comment, could you PTAL?